bricks/basic-bricks/editors/general-drawer/index.tsx

```diff
@@ -0,0 +1,36 @@
+import type _React from "react";
+import { customEditors } from "@next-core/runtime";
```

Comment:
@SailorF @ningkehua @MJmajiong v2 用法不一样，不要去用 v3 的包
```suggestion
import { getRuntime } from "@next-core/brick-kit";

getRuntime().customEditors.define(...)
```